### PR TITLE
doc: translate create a new react app

### PR DESCRIPTION
**Pull request by fabianlee1211**
This PR is ready for review!

### CODE REVIEW

File: content/docs/create-a-new-react-app.md [hunk 1]
```diff
@@ -1,94 +1,94 @@
...
+title: 建立全新的 React 應用程式
 permalink: docs/create-a-new-react-app.html
 redirect_from:
   - "docs/add-react-to-a-new-app.html"
 prev: add-react-to-a-website.html
 next: cdn-links.html
 ---
 
-Use an integrated toolchain for the best user and developer experience.
+使用綜合的 toolchain 以得到最佳的使用者和開發者體驗。
```
Comment (neighborhood999) [inline content/docs/create-a-new-react-app.md]:
```suggestion
使用整合的 toolchain 以得到最佳的使用者和開發者體驗。
```

File: content/docs/create-a-new-react-app.md [hunk 2]
```diff
@@ -1,94 +1,94 @@
...
 ```
 
->Note
+>提示
 >
->`npx` on the first line is not a typo -- it's a [package runner tool that comes with npm 5.2+](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b).
+>`npx` 不是拼寫錯誤 —— 它是一個 [npm 5.2+ 附帶的 package 執行器](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b)。
 
-Create React App doesn't handle backend logic or databases; it just creates a frontend build pipeline, so you can use it with any backend you want. Under the hood, it uses [Babel](https://babeljs.io/) and [webpack](https://webpack.js.org/), but you don't need to know anything about them.
+Create React App 並不會處理 backend 邏輯或數據庫；它只會建立一個 frontend build pipeline，以便你配合任何 backend 來使用。基本上，它是使用 [Babel](https://babeljs.io/) 和 [webpack](https://webpack.js.org/)，但你不需要了解任何關於它們的細節。
```
Comment (neighborhood999) [inline content/docs/create-a-new-react-app.md]:
```suggestion
Create React App 並不會處理 backend 邏輯或資料庫；它只會建立一個 frontend build pipeline，以便你配合任何 backend 來使用。基本上，它是使用 [Babel](https://babeljs.io/) 和 [webpack](https://webpack.js.org/)，但你不需要了解任何關於它們的細節。
```

File: content/docs/create-a-new-react-app.md [hunk 3]
```diff
@@ -1,94 +1,94 @@
...
 
-* Scaling to many files and components.
-* Using third-party libraries from npm.
-* Detecting common mistakes early.
-* Live-editing CSS and JS in development.
-* Optimizing the output for production.
+* 擴大文件和 component 的規模。
+* 使用來至 npm 的第三方函式庫。
+* 偵測早期常見的錯誤。
+* 實時在開發環境裡編輯 CSS 和 JS。
```
Comment (neighborhood999) [inline content/docs/create-a-new-react-app.md]: Maybe `Live-editing` keep the original term?

Comment (fabianlee1211) [reply content/docs/create-a-new-react-app.md]: I think translating `live-editing` wouldn't distort the meaning. Unlike verbs like `rendering` which might cause some technical misunderstanding after translation.

Comment (neighborhood999) [reply content/docs/create-a-new-react-app.md]: Got it!

**Pull request merged**